src/ledger/database/rocksdb.zig

```diff
@@ -87,6 +87,12 @@ pub fn RocksDB(comptime column_families: []const ColumnFamily) type {
         pub fn count(self: *Self, comptime cf: ColumnFamily) Allocator.Error!u64 {
             const live_files = try self.db.liveFiles(self.allocator);
             defer live_files.deinit();
+            defer for (live_files.items) |file| {
+                self.allocator.free(file.column_family_name);
+                self.allocator.free(file.name);
+                self.allocator.free(file.start_key.?);
+                self.allocator.free(file.end_key.?);
```

Comment:
```suggestion
                file.deinit();
```

Reply:
Thanks! I'll apply this suggestion. 

I'm also curious why this leak did not show up until now. Got any ideas?

Reply:
[74a13e2](https://github.com/Syndica/sig/pull/396/commits/74a13e281a85d357ee4d9fc41ab2f16cd04f406c)